shell/platform/darwin/common/BUILD.gn

```diff
@@ -56,3 +59,26 @@ source_set("framework_shared") {
 
   deps = [ "//flutter/fml" ]
 }
+
+test_fixtures("flutter_channels_fixtures") {
+  fixtures = []
+}
+
+# Unit tests for channels.
+executable("framework_shared_unittests") {
```

Comment:
Shouldn't this use the same nomenclature, "common" instead of "shared"?

Reply:
That's a good point. I named it after the framework_shared target, but framework_common is probably the better name. Will update. Thanks!